### DISCUSSION TITLE
Tabby as VNC viewer or VNC Client

### DISCUSSION THREAD

**Discussion by MachoDrone** [Eugeny/tabby#9712] [General]
I don't believe I see a VNC viewer or client to compliment my SSH connections. Do you know if VNC in on the wish-list?
I use VNC to watch TTY1 and TTY2 as i find SSH more suitable for interacting directly with the server.

**Comment by Eugeny** [comment]
It's not planned. A plugin could in theory implement it as a custom tab though.

**Reply by MachoDrone** [reply]
i wish i was capable of helping with a VNC plugin.. tabby is so great and i can't find another way to see live text on TTY1 and TTY2. RealVNC UI is so clunky, especially when each server is on a different resolution. tabby tabs are so orderly.
